storage/client/src/main/java/com/azure/storage/blob/AppendBlobAsyncRawClient.java

```diff
@@ -66,26 +69,20 @@
      * @param headers          {@link BlobHTTPHeaders}
      * @param metadata         {@link Metadata}
      * @param accessConditions {@link BlobAccessConditions}
-     * @param context          {@code Context} offers a means of passing arbitrary data (key/value pairs) to an
-     *                         {@link com.azure.core.http.HttpPipeline}'s policy objects. Most applications do not need to pass
-     *                         arbitrary data to the pipeline and can pass {@code Context.NONE} or {@code null}. Each context object is
-     *                         immutable. The {@code withContext} with data method creates a new {@code Context} object that refers to
-     *                         its parent, forming a linked list.
      * @return Emits the successful response.
      * @apiNote ## Sample Code \n
      * [!code-java[Sample_Code](../azure-storage-java/src/test/java/com/microsoft/azure/storage/Samples.java?name=append_blob "Sample code for AppendBlobAsyncRawClient.create")] \n
      * For more samples, please see the [Samples file](%https://github.com/Azure/azure-storage-java/blob/master/src/test/java/com/microsoft/azure/storage/Samples.java)
      */
     public Mono<AppendBlobsCreateResponse> create(BlobHTTPHeaders headers, Metadata metadata,
```

Comment:
What's the difference between Mono<Response<AppendBlobItem>and Mono<AppendBlobsCreateResponse? This seems more confusing than just picking one.

Reply:
This is the raw client hidden from users and to be removed after preview 1.